packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -143,88 +156,94 @@ String addServiceExtensions() {
                     });
                   }
                   additionalJson['constraints'] = constraintsProperty;
+                } catch (e) {
+                  // Not laid out yet.
                 }
                 if (renderObject is RenderBox) {
                   additionalJson['size'] = <String, Object>{
                     'width': renderObject.size.width.toString(),
                     'height': renderObject.size.height.toString(),
                   };
 
-                  final ParentData parentData = renderObject.parentData;
+                  final ParentData parentData =
+                      renderObject.parentData as ParentData;
                   if (parentData is FlexParentData) {
-                    additionalJson['flexFactor'] = parentData.flex;
+                    additionalJson['flexFactor'] = parentData.flex as int;
                     additionalJson['flexFit'] =
                         describeEnum(parentData.fit ?? FlexFit.tight);
                   }
                 }
               }
-              return additionalJson;
-            }),
-      );
-    }
-    return Future.value(<String, Object>{
+            }
+            return additionalJson;
+          }),
+    ) as Map<String, dynamic>;
+    return Future.value(<String, dynamic>{
       'result': result,
     });
   }
 
   Future<Map<String, dynamic>> setFlexFit(Map<String, String> parameters) {
-    final String id = parameters['id'];
-    final FlexFit flexFit =
-        toEnumEntry<FlexFit>(FlexFit.values, parameters['flexFit']);
+    final String id = parameters['id'] as String;
```

Comment:
Same question `final String` and `as String` only one needed?

Reply:
answered lower down.